dev/docker/compose

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -eou pipefail
 
-docker-compose -f dev/docker/docker-compose.yml -p "libxmtp" "$@"
+docker compose -f dev/docker/docker-compose.yml -p "libxmtp" "$@"
```

Comment:
The GH action runner has been upgraded and no longer has a separate `docker-compose` command. It's now a subcommand of `docker` only.